src/app/(app)/attendance/mark/page.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { CurrentAttendance } from '../components/CurrentAttendance'
+import { getLoggedInUser } from '@/utils/getLoggedInUser'
+
+const Page = async() => {
+  const loginUser = await getLoggedInUser();
```

Comment:
This is correct. @muhammad-bilal-ali-khokhar it's worth investigating why in client component we've login user is undefined at first.